.travis.yml

```diff
@@ -51,7 +51,7 @@ install:
   # Ensure pip and nose will install within the conda env
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip nose numpy matplotlib pandas
   - source activate test-environment
-  - pip install coverage codecov                           # code coverage; see other projects on how to install w/Miniconda
```

Comment:
This addition works well locally and on travis.  Travis shows the coverage reports for every build environment whereas before reports were not shown. I just need to exclude certain files from the coverage report.  I understand this can be done with .coveragerc.